python/bucketbase/namedlock.py

```diff
@@ -0,0 +1,53 @@
+import threading
```

Comment:
The filename `namedlock` doesn't follow the Python naming convention, and the class name. It should be `named_lock_manager`